src/main/resources/hudson/plugins/git/GitTool/global.jelly

```diff
@@ -31,5 +31,8 @@ THE SOFTWARE.
                      addCaption="${%Add Git}" hasHeader="true"
                      deleteCaption="${%Delete Git}"/>
     </f:entry>
+    <f:entry title="${%Git default timeout in minutes}" >
```

Comment:
Is it intentional that the same default timeout value might eventually be used for command line git, JGit, and the upcoming JGit with apache http authentication?

The current timeout only applies to command line git, and I think it makes sense that a global default should apply across all git implementations.